.github/workflows/publish-commit.yaml

```diff
@@ -0,0 +1,71 @@
+# Publishes main-branch's commits to pkg.pr.new
+# PRs can be published by commenting `/pkg-pr-new` in the PR
+
+name: Publish Any Commit
+
+on:
+  push:
+    branches:
+      - main
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    if: github.repository == 'stackblitz/tutorialkit' && (github.event_name == 'push' || github.event.issue.pull_request && startsWith(github.event.comment.body, '/pkg-pr-new'))
+    runs-on: ubuntu-latest
+
+    steps:
+      - if: github.event.issue.pull_request
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const user = context.payload.sender.login
+            console.log(`Validate user: ${user}`)
+
+            let hasTriagePermission = false
+            try {
+              const { data } = await github.rest.repos.getCollaboratorPermissionLevel({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                username: user,
+              });
+              hasTriagePermission = data.user.permissions.triage
+            } catch (e) {
+              console.warn(e)
+            }
+
+            if (hasTriagePermission) {
+              console.log('Allowed')
+              await github.rest.reactions.createForIssueComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                comment_id: context.payload.comment.id,
+                content: '+1',
+              })
+            } else {
+              console.log('Not allowed')
+              await github.rest.reactions.createForIssueComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                comment_id: context.payload.comment.id,
+                content: '-1',
+              })
+              throw new Error('not allowed')
+            }
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/setup-and-build
+
+      - name: Publish to pkg.pr.new
+        run: >
+          pnpm dlx pkg-pr-new@0.0 publish --compact --pnpm
+          ./packages/astro
+          ./packages/components/react
+          ./packages/runtime
+          ./packages/theme
+          ./packages/types
```

Comment:
@Aslemammad I would rather use `--filter='@tutorialkit/*'` here - is it possible? There are some packages in `./packages/` that we don't want to publish - only the ones that have `@tutorialkit/*` in the name.

Similar to:

https://github.com/stackblitz/tutorialkit/blob/74e052701934fbbff9d8bcd06dafc88c58b83667/.github/workflows/publish-release.yaml#L29-L34

Reply:
i mean, we support globs, you can see the cli/index.ts file in pkg.pr.new! So if that's valid glob, it'd work.

Reply:
Is it glob for filenames and paths, or for `name`'s in `package.json`?